src/app/project/list/list-item/list-item.component.ts

```diff
@@ -36,14 +39,43 @@ export class ListItemComponent implements OnInit {
 
     @Output() refreshChildren: EventEmitter<ListNode[]> = new EventEmitter<ListNode[]>();
 
+    // permissions of logged-in user
+    session: Session;
+    sysAdmin = false;
+    projectAdmin = false;
+
+    // project data
+    project: ReadProject;
```

Comment:
same thing here as in the `list-item-form` component

Reply:
https://github.com/dasch-swiss/dsp-app/pull/825/commits/ff8b5a85e3d05f019d874791b5de77a05edb7572